core/src/test/java/nl/nn/adapterframework/doc/DocWriterNewIntegrationTest.java

```diff
@@ -2,37 +2,69 @@
 
 import java.io.BufferedWriter;
 import java.io.File;
+import java.io.FileInputStream;
 import java.io.FileWriter;
 import java.io.IOException;
 import java.io.Writer;
 
+import javax.xml.XMLConstants;
+import javax.xml.transform.sax.SAXSource;
+import javax.xml.validation.Schema;
+import javax.xml.validation.SchemaFactory;
+import javax.xml.validation.Validator;
+
+import org.apache.logging.log4j.Logger;
 import org.junit.Ignore;
 import org.junit.Test;
+import org.xml.sax.InputSource;
 
 import nl.nn.adapterframework.doc.model.FrankDocModel;
 import nl.nn.adapterframework.doc.model.FrankElement;
 import nl.nn.adapterframework.doc.model.FrankElementStatistics;
+import nl.nn.adapterframework.doc.model.XsdVersion;
+import nl.nn.adapterframework.util.LogUtil;
 
-@Ignore("Test takes a long time to run, and gives little information")
 public class DocWriterNewIntegrationTest {
```

Comment:
I deliberately removed the ignore. I added to the tests that the XSD being written is also tried on an example configuration. I think it is very useful to test during the build that our FF! code fits in a usable XML schema.